kuma/wiki/tests/test_models.py

```diff
@@ -333,6 +332,39 @@ def test_redirect_url_works_for_home_path(self):
         d = document(is_redirect=True, html=html)
         eq_(href, d.get_redirect_url())
 
+    @mock.patch('kuma.wiki.tasks.set_share_url')
+    def test_get_share_url_empty(self, _mock):
+        doc = document(title='test', share_url=None)
```

Comment:
that's odd, the document helper doesn't save the Document by default, so I'm not sure why the mock is being called. also, mind not using a underscored variable names here?